pkg/sdk/databases.go

```diff
@@ -34,7 +34,6 @@ type Databases interface {
 	Show(ctx context.Context, opts *ShowDatabasesOptions) ([]Database, error)
 	ShowByID(ctx context.Context, id AccountObjectIdentifier) (*Database, error)
 	Describe(ctx context.Context, id AccountObjectIdentifier) (*DatabaseDetails, error)
-	Use(ctx context.Context, id AccountObjectIdentifier) error
```

Comment:
hmmm, why we are removing it? Does it bother us to have it in both places? Will we remove similar functions from other resource too?

Reply:
It wasn't used anywhere and I assumed we wanted to limit the number of ways to perform a certain action, so I just removed it since we already have `client.Sessions.UseDatabase`.

Reply:
let's leave it for now, so that we don't have similar discussions in every other object